Functions/xASL_adm_GetnSessions.m

```diff
@@ -0,0 +1,57 @@
+function [nSessions, bSessionsMissing] = xASL_adm_GetnSessions(x)
```

Comment:
wonder if we should call this `xASL_adm_GetPopulationSessions` to clarify that this is for the population module/folder only

Reply:
Agreed